### PR TITLE
Fix dual stack mode issue for udp socket with IPv6

### PR DESCRIPTION
**Pull request by repeatedly**
See: https://github.com/fluent/fluentd/pull/2781

### CODE REVIEW

File: spec/socket_manager_spec.rb [hunk 1]
```diff
@@ -155,7 +155,29 @@
...
-    end if (TCPServer.open("::1",0) rescue nil)
+    end
+
+    unless ServerEngine.windows?
+      context 'using ipv4/ipv6' do
+        it 'can bind ipv4/ipv6 together' do
+          server = SocketManager::Server.open(server_path)
+          client = ServerEngine::SocketManager::Client.new(server_path)
+
+          p :foo
```
Comment (ganmacs) [inline spec/socket_manager_spec.rb]: Looks unnecessary.

Comment (repeatedly) [reply spec/socket_manager_spec.rb]: Ah, my mistake.

**Comment by ganmacs** [comment]
CI seems to be failing.
 https://travis-ci.org/treasure-data/serverengine/builds/639306379?utm_source=github_status&utm_medium=notification

**Comment by repeatedly** [comment]
fixed

**Comment by repeatedly** [comment]
If no problem, I will merge the patch.

**Pull request merged**